NSS-Day-Cohort-53/tabloid-full-stack-always-puffy-impalas
# Sh view categories

#### Changes Made
1. Added file `Category.cs` class to `models` directory.
2. Added `CategoryRepository.cs` and `ICategoryRepository.cs` to `repositories` directory.
3. Added `categoryManager.js` in `modules` folder within `client` directory.
4. Added `Category.js` , and `CategoryList.js` in `components` folder within `client` directory
5. Modified `Header.js` to include a link to Categories page
6. Modified `ApplicationViews.js` to include a route to Categories page. 
7. Merged new main, fixed merge conflicts.

​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout SH-ViewCategories
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
8. Test app functionality.
    > When a user navigates to the Categories link, they should see a list of all categories
9. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
